fem/complex_fem.hpp

```diff
@@ -175,13 +187,35 @@ class SesquilinearForm
 private:
    ComplexOperator::Convention conv;
 
+   /**
+    * This member allows one to specify what should be done
+    * to the diagonal matrix entries and corresponding RHS
+    * values upon elimination of the constrained DoFs.
+    */
+   mfem::Matrix::DiagonalPolicy diag_policy = mfem::Matrix::DIAG_ONE;
+
    BilinearForm *blfr;
    BilinearForm *blfi;
 
+   /* These methods check if the real/imag parts of the
+   sesqulinear form are not empty */
+   bool RealInteg();
+   bool ImagInteg();
+
 public:
    SesquilinearForm(FiniteElementSpace *fes,
                     ComplexOperator::Convention
                     convention = ComplexOperator::HERMITIAN);
+   /** @brief Create a BilinearForm on the FiniteElementSpace @a f, using the
+       same integrators as the BilinearForms @a bfr and @a bfi .
+
+       The pointer @a fes is not owned by the newly constructed object.
+
+       The integrators are copied as pointers and they are not owned by
+       the newly constructed BilinearForm. */
```

Comment:
BilinearForm should be SesquilinearForm.

Reply:
Done!

Reply:
I still see BilinearForm on line 215.

Reply:
Done!